src/main/kotlin/app/batch/S3DirectoryReader.kt

```diff
@@ -62,6 +62,10 @@ class S3DirectoryReader(private val s3Client: AmazonS3, private val s3Utils: S3U
                 request.continuationToken = results.nextContinuationToken
             } while (results != null && results.isTruncated)
 
+            logger.info("s3 object count",
```

Comment:
Can we start with a capital please? Sorry...

Reply:
done.